src/rsz/src/RepairDesign.hh

```diff
@@ -107,6 +107,12 @@ class RepairDesign : dbStaState
                  double cap_margin);
   void repairClkNets(double max_wire_length);
   void repairClkInverters();
+  void printViolationCounters(bool invalidate_driver_vertices,
```

Comment:
I believe `reportViolationCounters` is a better name. Generally `report` over `print` is a better option.